pyproject.toml

```diff
@@ -18,121 +18,100 @@ convention = "numpy"
 [tool.ruff.lint.flake8-copyright]
 author = "- Pruna AI GmbH"
 
+[tool.uv]
+extra-index-url = ["https://prunaai.pythonanywhere.com/"]
+
```

Comment:
Why are we adding this?

Reply:
Hi David,

It's coming from here:
https://github.com/PrunaAI/pruna/blob/01a57b2ada6b56893fda0c232584955b7b8912ed/pyproject.toml#L40-L43

It configures an additional Python package source (repository) for the project